dxtbx/model/__init__.py

```diff
@@ -396,6 +397,12 @@ def profiles(self):
     ''' Get a list of the unique profile models (includes None). '''
     from libtbx.containers import OrderedSet
     return list(OrderedSet([e.profile for e in self]))
+  
+  def scaling_models(self):
+    ''' Get a list of the unique scaling models (includes None). '''
+    from libtbx.containers import OrderedSet
```

Comment:
is the order relevant? why not simply _return list(set(e.profile for e in self))_?
If order is actually relevant please remove all _import OrderedSet_ from file and add a single import to the top.

Reply:
Order may be relevant, the functions were intended to return the unique models in the order they were encountered in the experiment list so I think this method is fine. Also I prefer having imports inside functions rather than in the file scope so I think this is fine.

Reply:
imports inside functions not really sensible here though. Imo that should only be done in a case where lazy importing actually makes a difference. I do not think this is the case here?
I would put _import libtbx.containers at the top, then the dependencies are clear to the reader at a glance, as is the origin of the name.